vars/folioEurekaApp.groovy

```diff
@@ -0,0 +1,41 @@
+import org.folio.Constants
+import org.folio.utilities.Logger
+
+
+Logger logger = new Logger(this, 'folioEurekaApp')
+
+def generateAppPlatformMinimalDescriptor() {
```

Comment:
Maybe it should be on method named generateAppDescriptor and accept aruments. If we need this argument be validate or static values, you could use eval or any other approach